packages/api/models/executions.js

```diff
@@ -141,7 +142,7 @@ class Execution extends Manager {
    */
   _getMutableFieldNames(record) {
     if (record.status === 'running') {
-      return ['createdAt', 'updatedAt', 'timestamp', 'originalPayload'];
+      return ['updatedAt', 'timestamp', 'originalPayload'];
```

Comment:
Admittedly this change is outside the scope of the actual ticket and unnecessary. @nemreid is actually the one who noticed it first. Basically, for execution records, there's no reason that the `createdAt` timestamp should ever change. Because even in out of order processing for an execution where the `running` event came after the `completed` event, we wouldn't want to change the `createdAt` time

Does that make sense?